.github/workflows/code-quality.yml

```
@@ -81,28 +108,23 @@ jobs:
       - uses: actions/checkout@v7
         with:
           fetch-depth: 0
+          persist-credentials: false
+      - uses: nrwl/nx-set-shas@v4
+        with:
+          main-branch-name: master
       - uses: ./.github/actions/setup-node-pnpm
-      - name: Lint and format check across all TS/JS/JSON/CSS files
+      - name: Lint and format check via nx (cache:true)
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:4:218: Double quote to prevent globbing and word splitting

(shellcheck)